packages/shared/src/components/checklist/SquadChecklistCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactElement } from 'react';
+import { ChecklistCard } from './ChecklistCard';
+import { useSquadChecklistSteps } from '../../hooks/useSquadChecklistSteps';
+import { Squad } from '../../graphql/sources';
+import usePersistentContext from '../../hooks/usePersistentContext';
+import { SQUAD_CHECKLIST_VISIBLE_KEY } from '../../lib/checklist';
+import InteractivePopup, {
+  InteractivePopupPosition,
+} from '../tooltips/InteractivePopup';
+import useSidebarRendered from '../../hooks/useSidebarRendered';
+import { Modal } from '../modals/common/Modal';
+import { ModalKind } from '../modals/common/types';
+
+const SquadChecklistCard = ({ squad }: { squad: Squad }): ReactElement => {
+  const { steps } = useSquadChecklistSteps({ squad });
+  const [isChecklistVisible, setChecklistVisible] = usePersistentContext(
+    SQUAD_CHECKLIST_VISIBLE_KEY,
+    false,
+  );
+  const { sidebarRendered } = useSidebarRendered();
+
+  if (!isChecklistVisible) {
+    return null;
+  }
+
+  const checklistElement = (
```

Comment:
would it make any difference defining this after the modal return as it's not used if that's the case?

Reply:
It is used in both cases. I just use Modal vs InteractivePopup as wrappers.